mappings/net/minecraft/world/World.mapping

```diff
@@ -85,6 +85,9 @@ CLASS net/minecraft/class_1937 net/minecraft/world/World
 		ARG 2 entity
 		ARG 3 direction
 	METHOD method_27982 isDebugWorld ()Z
+		COMMENT Checks if this world is a debug world.
+		COMMENT
+		COMMENT <p>Debug worlds are not modifiable and are typically meant for development and debug use only.
```

Comment:
maybe just link to minecraft wiki

Reply:
This would be the relevant link:
https://minecraft.gamepedia.com/Debug_mode